src/Controllers/ElementalAreaController.php

```diff
@@ -45,132 +47,153 @@ public function getClientConfig()
         $clientConfig = parent::getClientConfig();
         $clientConfig['form']['elementForm'] = [
             'schemaUrl' => $this->Link('schema/elementForm'),
-            'saveUrl' => $this->Link('api/saveForm'),
-            'saveMethod' => 'post',
-            'payloadFormat' => 'json',
-            'formNameTemplate' => sprintf(static::FORM_NAME_TEMPLATE ?? '', '{id}'),
+            'formNameTemplate' => sprintf(static::FORM_NAME_TEMPLATE, '{id}'),
         ];
-
         // Configuration that is available per element type
         $clientConfig['elementTypes'] = ElementTypeRegistry::generate()->getDefinitions();
-
         return $clientConfig;
     }
 
     /**
-     * @param HTTPRequest|null $request
+     * Used for both:
+     * - GET requests to get the FormSchema via getElementForm() called from LeftAndMain::schema()
+     * - POST Requests to save the Form. Will be handled by to FormRequestHandler::httpSubmission()
+     * /admin/linkfield/elementForm/<ElementID>
+     *
      * @return Form
-     * @throws HTTPResponse_Exception
      */
     public function elementForm(HTTPRequest $request = null)
     {
-        // Get ID either from posted back value, or url parameter
-        if (!$request) {
+        $request = $request ?: $this->getRequest();
+        $id = $request->param('ItemID');
+
+        // Respect previous behaviour
+        // This should just be removed later and so that it 404's below when BaseElement::get()->byID($id) fails
```

Comment:
```suggestion
```
This is no longer valid due to the decision in https://github.com/silverstripe/silverstripe-elemental/pull/1150#discussion_r1531453680

Reply:
I think the comment still makes sense. The logic to return a 400 will be removed in PR to CMS 6 so it returns a 404 in this scenario rather than a 400.

PR is [https://github.com/creative-commoners/silverstripe-elemental/pull/2/files#diff-942115e4b8f6030e4d72ebc2b3a0772ec65e5dfcd08fbd0e677c70d1231daf28L74](here) - note it needs a rebase the branch it's from is very out of date